README.md

```diff
@@ -686,6 +686,27 @@ such behavior for `hidden` or `required`; use `accessor.hidden` and
 ``accessor.required` in your rendering code to determine whether a field wants
 to be hidden or required.
 
+## Warnings
+
+mstform has a hook which allows you to include `warning` messages in the fields.
+Warnings are similar to errors, however, the philosophy behind it is that you
```

Comment:
a "philosophy" is a bit big. "Warnings are similar to errors, but don't make the form invalid. The idea is that you can show warnings for certain fields in your form as a notification to the user". Or somethingl ike that.